### DISCUSSION TITLE
Help Setting up VISX for my react project

### DISCUSSION THREAD

**Discussion by knightswatch3** [airbnb/visx#1146] [Q&A]
import {Group} from "@visx/group" ;
import { hierarchy, Tree } from "@visx/hierarchy";
import { LinearGradient } from "@visx/gradient";

I'm new to the visx library and I am trying to import the needed components in my react(javascript) project. For which I receive the following errors:

ERROR in ./src/client/view/routes/Tasks/routes/Edit/components/EditNewGraph/EditNewGraph.js
Module not found: Error: Can't resolve '@visx/gradient' in '/root/T1/src/client/view/routes/Tasks/routes/Edit/components/EditNewGraph'
 @ ./src/client/view/routes/Tasks/routes/Edit/components/EditNewGraph/EditNewGraph.js 32:0-48

I don't think I have to tweak anything on the webpack configuration. 

But I am running out of ideas of where should I be looking at the errors.

TIA.

My version of webpck - 4.44.1
React - 16.x
node v14.0.0

**Comment by hshoff** [comment]
Hi @SatishChandraPalakoti,

Could you try running `npm install --save @visx/gradient` or `yarn add @visx/gradient`? Looking at the error, it appears it's not installed/not in `/node_modules`.
